modules/op-release-notes-1-16.adoc

```diff
@@ -0,0 +1,594 @@
+        config-defaults-triggers: 
+          data:
+            default-run-as-non-root: "true"
+            default-run-as-user: "65532"
+            default-run-as-group: "65532"
+            default-fs-group: "65532"
+----
++
+Optionally, you can set the values of the `default-run-as-user` and `default-run-as-group` parameters to configure the numeric user ID and group ID for running the event listeners in containers; {pipelines-shortname} sets these values in the pod security context and container security context for event listeners. If you use empty values, the default user ID and group ID of `65532` are used.
++
+You can also set the `default-fs-group` parameter to define the `fsGroup` value for the pod security context, which is the group ID that the container processes use for the file system. If you use an empty value, the default group ID of `65532` is used.
+
+* With this update, in pipelines triggers, the `EventListener` pod template now includes `securityContext` settings. Under these settings, you can configure `seccompProfile`, `runAsUser`, `runAsGroup`, and `fsGroup` parameters when the `el-security-context` flag is set to `true`. 
+
+[id="web-console-new-features-1-16_{context}"]
+=== Web console
+
+* Before this release, when using the web console, you could not see the timestamp for the logs that {pipelines-shortname} created. With this update, the web console includes timestamps for all logs for {pipelines-shortname}.
+
+* With this update, the pipeline run and task run *list* pages in the web console now have a filter for the data source, such as `k8s` and `TektonResults API`.
+
+* Before this update, when using the web console in the *Developer* perspective, you could not specify the timeout for pipeline runs. With this update, you can set a timeout while starting the pipeline run in the *Developer* perspective of the web console. 
+
+* Before this update, the *Overview* pipeline dashboard only appeared when {tekton-results} was enabled. All the statistics came from only the Results API. With this update, the *Overview* pipeline dashboard is available regardless of whether {tekton-results} is enabled or not. When {tekton-results} is disabled, you can use the dashboard to see the statistics for objects in the cluster.
+
+* With this update, the sample pipelines displayed in the web console use the `v1` version of the {pipelines-shortname} API.
+
+[id="cli-new-features-1-16_{context}"]
+=== CLI
+
+* With this update, you can use the `tkn customrun delete <custom_run_names>` command to delete one or more custom runs.
+
+* With this update, when you run a `tkn <resource> list` command with the `-o` YAML flag, the listed resources are now separated with `---` separators to enhance readability of the output.
+
+[id="pac-new-features-1-16_{context}"]
+=== {pac}
+
+* With this update, if you create two `PipelineRun` definitions with the same name, {pac} logs an error and does not run either of these pipeline runs.
```

Comment:
@mramendi i think in the google doc there was more info and its not there here
basically we want to include below info
```
Redesign and Enhancement of PAC Resolver to cover few scenarios like https://github.com/openshift-pipelines/pipelines-as-code/pull/1762
* PAC will not resolve Pipelines unnecessarily which are not part of the PipelineRuns filtered for the event
* Now you can use different version of same Task in different PipelineRuns/Pipelines.
```
ref : https://github.com/openshift-pipelines/pipelines-as-code/releases/tag/v0.28.0

Reply:
Hello @savitaashture , I deleted the entry you are mentioning, because it was a duplicate of these two:

* Before this update, when using {pac}, if your `.tekton` directory contained a pipeline that was not referenced by any `PipelineRun` definition triggered in the event, {pac} attempted to fetch all the required tasks for that pipeline even though it was not run. With this update, {pac} does not try to resolve pipelines that are not referenced in any pipeline run triggered by the current event.

* Before this update, when using {pac}, if different `PipelineRun` definitions used the same task name but different versions, for example when fetching tasks from {tekton-hub},the wrong version was sometimes triggered, because {pac} used the same task version for all pipeline runs. With this update, {pac} triggers the correct version of the referenced task.

Please correct me if that assumption was wrong, but I though these two and the ones you mentioned are referring to the same thing. Thank you!

Reply:
Yes yes i m referring these two 
i see they are covered 
LGTM

Reply:
Great, thanks!